public/js/module.js

```diff
@@ -155,7 +155,9 @@
                 ].join('&');
 
                 var $container = $target.closest('.container');
-                icinga.loader.loadUrl(actionUrl, $container, data, 'POST');
+                icinga.loader.loadUrl(actionUrl, $container, data, 'POST')
```

Comment:
```suggestion
                this.module.icinga.loader.loadUrl(actionUrl, $container, data, 'POST')
```

Reply:
It works fine and is not a new change, so it can be left as it is.

Reply:
I understand that it works fine. But I discussed this yesterday with @nilmerg, He mentioned that it is preferrable to access `icinga` property through  `this.module` instead of directly accessing the global variable `icinga`.